### PR TITLE
Added community contribution work on R vs Tableau.

### PR DESCRIPTION
**Pull request by AmoghM**
This is a joint work of team CC51 consisting of 

- Amogh Mishra (am5323)

- Pritam Biswas (pb2796)

Changes requested:
1. Added r_vs_tableau.rmd
2. Added images, data and tableau workbook.

### CODE REVIEW

**Comment by kmsingh-git** [comment]
Getting an Error: @bojum Can you take a look? 

Here's the stacktrace from the build

Quitting from lines 61-72 (r_vs_tableau_plots.Rmd) 
Error in file(file, "rt") : cannot open the connection
Calls: local ... withVisible -> eval -> eval -> read.csv -> read.table -> file
Execution halted
Error in Rscript_render(f, render_args, render_meta) : 
  Failed to compile r_vs_tableau_plots.Rmd
Calls: <Anonymous> ... render_new_session -> tryCatch -> tryCatchList -> Rscript_render
Execution halted
The command "Rscript -e 'bookdown::render_book("index.Rmd")'" exited with 1.
cache.2
store build cache

**Comment by bojum** [comment]
@kmsingh-git The error indicates that there is a file that can't be open somewhere in line 61-72. Usually, this happens when contributors moved their files into resources/<project_name>/ and forgot to change file paths in their Rmd files.

In this case, they need to fix line 56: MyData <- read.csv(file="resources/air_quality.csv", header=TRUE, sep=","). (Hmmm not sure why line numbers are off...)

**Comment by kmsingh-git** [comment]
Oh yea that's what I thought! but on lines 61-72 there was no such thing :P Thanks @bojum. I'll file it as a bug and request the team to fix it then?

**Comment by bojum** [comment]
Request a change and label it as `changes requested`. `bug` label is for PRs that fail the build test.

File: r_vs_tableau_plots.Rmd [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+chart_link = api_create(p, filename="choropleth-world")
+```
+
+The interactive choropleth map is at location https://plot.ly/~biswaspritam1993/3/
+
+This is the NO2 and NOx concentrations in air line plot
+```{r comment=NA}
+library(ucidata)
+
+MyData <- read.csv(file="resources/air_quality.csv", header=TRUE, sep=",")
```
Comment (kmsingh-git) [inline r_vs_tableau_plots.Rmd]: There are errors in compiling this file. look at the Comments on Pull Request for a stack trace.

**Review by kmsingh-git — CHANGES_REQUESTED** [review]

File: r_vs_tableau_plots.Rmd (hunk 1 above)
Comment (AmoghM) [reply r_vs_tableau_plots.Rmd]: @kmsingh-git Thanks for pointing it out. We missed this out while uploading it to git.

**Review by kmsingh-git — APPROVED** [review]

**Comment by kmsingh-git** [comment]
All Checks passed after author changed code. Should be good to merge now

**Comment by bojum** [comment]
🎉 The work has been merged! Thanks for your contribution!

**Pull request merged**